src/app/modules/shared-components/components/message/message.component.ts

```diff
@@ -0,0 +1,27 @@
+import { Component, OnInit, Input } from '@angular/core';
+import { Message } from 'primeng/api';
+
+@Component({
+  selector: 'alg-message',
+  templateUrl: './message.component.html',
+  styleUrls: ['./message.component.scss']
+})
+export class MessageComponent implements OnInit {
+
+  @Input() type: 'success' | 'info' | 'error';
+  @Input() label: string;
+  @Input() closable = true;
+
+  msgs : Message[] = [];
+
+  constructor() { }
+
+  ngOnInit() {
```

Comment:
It means that if the input changes (`label`), the message will not be updated. You should do in in ngOnChange (and change the class `implements..`).
Beware that in this case, "push"ing is not correct.